src/js/components/Menu/stories/CustomThemed/Themed.tsx

```diff
@@ -60,10 +60,16 @@ const customBreakpoints: ThemeType = {
     },
   },
   menu: {
+    background: 'light-3',
     icons: {
       down: FormDown,
       up: FormUp,
     },
+    button: {
+      color: 'brand',
+      style: { margin: '6px' },
```

Comment:
Can you share a visual on why `margin` and not `pad` somewhere? Is the idea to create space around the menu items? If so, I wonder if `theme.menu.itemContainer: BoxType` might be more consistent, like `theme.notification.textContainer`.

Reply:
I used `margin` just for the easy example, I'll replace it with a font style to get closer to my use case where font-weight should behave differently on the menu button.